blocks/small-manual-promo-block/features/small-manual-promo/default.jsx

```diff
@@ -1,23 +1,81 @@
 import React from "react";
 import PropTypes from "@arc-fusion/prop-types";
 import { useFusionContext } from "fusion:context";
-import { LazyLoad, isServerSide } from "@wpmedia/engine-theme-sdk";
-import { imageRatioCustomField } from "@wpmedia/resizer-image-block";
-import { SmallPromoPresentation } from "@wpmedia/shared-styles";
+import {
+	formatURL,
+	Heading,
+	HeadingSection,
+	Image,
+	isServerSide,
+	Link,
+	MediaItem,
+	Stack,
+} from "@wpmedia/arc-themes-components";
+import { LazyLoad } from "@wpmedia/engine-theme-sdk";
 
-const SmallManualPromo = ({
-	customFields = { showImage: true, showHeadline: true, imageRatio: "3:2" },
-}) => {
+const BLOCK_CLASS_NAME = "b-small-manual-promo";
+
+const SmallManualPromo = ({ customFields }) => {
+	const {
+		headline,
+		imagePosition,
+		imageRatio,
+		imageURL,
+		lazyLoad,
+		linkURL,
+		newTab,
+		showHeadline,
+		showImage,
+	} = customFields;
 	const { isAdmin } = useFusionContext();
-	const shouldLazyLoad = customFields?.lazyLoad && !isAdmin;
-	if (customFields.lazyLoad && isServerSide() && !isAdmin) {
-		// On Server
+	const shouldLazyLoad = lazyLoad && !isAdmin;
+	if (shouldLazyLoad && isServerSide()) {
 		return null;
 	}
 
+	const PromoImage = () =>
```

Comment:
We have lost the inline ability to use the integrated photo search selector

Reply:
Do you still see the inline button as shown here?
<img width="277" alt="Google Chrome-2022-06-20-19-32 47" src="https://user-images.githubusercontent.com/868127/174660664-510bbe70-c583-4464-b5b0-06238b0074eb.png">

Reply:
No. Happen to know the cause or means to resolve? If not I can look into it. Thanks!
<img width="232" alt="image" src="https://user-images.githubusercontent.com/452134/174664155-e619bc7b-7954-4d98-a62c-db8d348ff87f.png">

Reply:
TYVM! Will give this a try!